src/utils/validation/registration/registrationValidation.ts

```diff
@@ -87,14 +87,7 @@ export const registrationValidationSchema = Yup.object<YupShape<Registration>>({
       }
     }),
   }),
-  associatedArtifacts: Yup.array()
-    .of(associatedFileValidationSchema)
-    .when('entityDescription', ([entityDescription]: EntityDescription[], schema) =>
-      entityDescription.reference?.doi
-        ? schema.min(0)
-        : schema.min(1, registrationErrorMessage.associatedArtifactRequired)
```

Comment:
verdien for `registrationErrorMessage.associatedArtifactRequired` kan fjernes også, sammen med oversettelsen der